arches/app/src/components/ControlledListManager/ControlledListEditor.vue

```diff
@@ -0,0 +1,220 @@
+<script setup lang="ts">
+import arches from "arches";
+import Cookies from "js-cookie";
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+import Splitter from "primevue/splitter";
+import SplitterPanel from "primevue/splitterpanel";
+import { useToast } from "primevue/usetoast";
+
+import ControlledListSplash from "@/components/ControlledListManager/ControlledListSplash.vue";
+import ItemEditor from "@/components/ControlledListManager/ItemEditor.vue";
+import ListCharacteristics from "@/components/ControlledListManager/ListCharacteristics.vue";
+import ListHeader from "@/components/ControlledListManager/ListHeader.vue";
+import SidepanelDataView from "@/components/ControlledListManager/SidepanelDataView.vue";
+import SpinnerIcon from "@/components/SpinnerIcon.vue";
+
+import type { Ref } from "vue";
+import type {
+    ControlledList,
+    ControlledListItem,
+    LanguageMap,
+} from "@/types/ControlledListManager.d";
+
+const lightGray = "#f4f4f4";
+const buttonGreen = "#10b981";
+const toast = useToast();
+const { $gettext, $ngettext } = useGettext();
+
+const props: {
```

Comment:
open for discussion here but my kneejerk response is to say props go under imports and above delcarations

Reply:
Also suggestible here, as I didn't even know you _could_ put them above the declarations! I guess my mind is used to "get the symbol before you use it", but I don't really have an opinion.